### PR TITLE
data: VidipGhosh

### PR DESCRIPTION
**Pull request by Vidip-Ghosh**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by sital002 — CHANGES_REQUESTED** [review]
Your filename must match your github username instead of ```data/VidipGhosh.json``` it should be ```data/Vidip-Ghosh.json```.
Please make the necessary changes to get your profile merged.

File: data/VidipGhosh.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Vidip Ghosh",
+  "bio": "B.Tech CSE'24 from Jamia Millia Islamia, New Delhi",
+  "tags": ["travelling","photography","exploring new technologies"]
```
Comment (sital002) [inline data/VidipGhosh.json:4]:
```suggestion
  "tags": ["travelling","photography","exploring new technologies"],
```
There is a comma missing here

**Comment by Vidip-Ghosh** [comment]
I've made the necessary changes. Please review.

**Review by sital002 — APPROVED** [review]
Looks good 👍
Welcome to Eddiehub community 🎉🎉

**Pull request merged**